web/vite.config.ts

```diff
@@ -98,7 +90,25 @@ export default ({ mode }: { mode: any }) => {
     },
     build: {
       manifest: true,
-      sourcemap: true
+      sourcemap: true,
+      rollupOptions: {
+        output: {
+          manualChunks (id: string) {
```

Comment:
This `manualChunks` is kinda weird, but without it, Vite by default only splits `index` and `vendor`, which isnt good enough to me.

Reply:
Yes, but to be honest, I could not understand well 🤓

And the `manualChunks` does not work without the plugin